CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 0.26.0 / 2024-05-08
+
+* [CHANGE] Improve generator parse error handling #1167
+
+* [ENHANCEMENT] generator: Add generator HELP override #1106
+* [ENHANCEMENT] Refactoring of Scrape process, fixing multiple module issues #1111
+* [ENHANCEMENT] Skip using an interactive terminal in "make docker-generate". #1113
+* [ENHANCEMENT] Add SNMPInflight metric #1119
+* [ENHANCEMENT] upgrading cyberpower mib to v2.11; support for ats #1124
+* [ENHANCEMENT] adding dell support #1125
```

Comment:
These also go under snmp.yml changes.

Reply:
fixed in commit 03a69f9d75d0b9e40a786dc8e60f97d51ffcbb97